### ISSUE TITLE
Pointing a variable after the variable 5000 doesn't work

### ISSUE THREAD

**Issue by BlisterBoy** [EasyRPG/Player#477]
Hi guys :) ! Here's a tricky bug that a friend found.
RPG Maker has recently increase to 9999 the number of switchs and variables, Easy RPG has done the same but it seems that the referencing system hasn't been changed.
For exemple : 
I do V[1] = 7000
But if I do V[V[1]] = 42 that just doesn't work, the variable 7000 isn't changed.
But it works if V[1] = 20.

NB : It works on RPG Maker
NB2 : If a directly do V[7000] = 42 in Easy RPG, it works

**Comment by BlisterBoy** [comment]
Fun fact, it was a trick used to bypass the 5000 variables restrictions in french community.

**Comment by fdelapena** [comment]
A question: is this working when you do \V[1] = 3000? Current code is not limited to 5000.

**Comment by Ghabry** [comment]
Referencing an incorrect variable will return a reference to a dummy-var. So accessing 0, 9999 and 7000 points to the same :).
I think the problem is that RPG_RT does on-the-fly resizing when you do a out-of-bounds access. So if you declared only 100 vars in the editor and you access 200 RPG_RT resizes the array to 200 elements. We don't... does anybody really depend on this? -.-

**Comment by BlisterBoy** [comment]
Unfortunatly... Yes hahaha. 
Please notice that this is not anymore an incorrect use of variables referencing because RPG Maker does support 99999 variables.

fdelapena : I'm currently on Mac OS, I'll tell you that and upload the test project when I'll go back to Windows (I'm writing a presentation of Easy RPG for french community ^^), which website do you use to upload test projects ?

**Comment by fdelapena** [comment]
@BlisterBoy do you mean projects for testing from the HTML5 player? they are uploaded by hand to the web server, if you have the game list for downloading and trying. They need a conversion from midi to ogg and minify assets for better performance. There are however some issues not fixed yet (e.g. black lower layer tileset)
You could also get the HTML5 player and put all games in another host. It requires to run a php script for the online file finder.

**Comment by BlisterBoy** [comment]
Excuse me I didn't explained well, I wanted to upload the testproject (google drive, mega or whatever) so you can test difference with RPG Maker Player and Easy RPG.

But thanks for the explanations, some frenchies ask me questions about the web port today ^^.

**Comment by carstene1ns** [comment]
You can use whatever hosting service you want, it is just important that it can be accessed without problems. As most of these one-click hosters will remove the files after some time, it might be better to use dropbox or google drive. Personally, I use my homepage ;-)

**Comment by BlisterBoy** [comment]
Okay I used Google Drive, here is the folder with the RPG Maker Player and the Easy RPG one.
https://drive.google.com/open?id=0BzyFBzg6Eyy5bW9iTjAyWkNSRUE&authuser=0

@fdelapena : Yep it works when referencing the variable 3000, but not 70000. I included this test in the test project :).

(PS : I've presented the project to french communities, frenchies are really excited and impatient about the project ^^)
